.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: ruby
 rvm:
-  - 1.9.2
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
+  - 2.1.10
+  - 2.2.9
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
```

Comment:
I'll let @kytrinyx make the final call here, but in the past we haven't wanted to drop support for the older Ruby versions.  approvals is a tool that is especially useful for getting test coverage around legacy code, and that legacy code might still be running on an older, unsupported Ruby version.

Reply:
An alternative approach to this could be to release a new version which drops support for Ruby versions which have reached end of life. Existing versions of `approvals` will still work.

Reply:
At any rate, I've restored the previous versions to the build matrix. 🙇

Reply:
> An alternative approach to this could be to release a new version which drops support for Ruby versions which have reached end of life. Existing versions of approvals will still work.

That might be good, actually. We can document in the README which version drops support.